pkg/load/agent_utils.go

```diff
@@ -0,0 +1,52 @@
+package load
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fsnotify/fsnotify"
+	log "github.com/sirupsen/logrus"
```

Comment:
nit: do we need the `log` name? most other places are fine with using the `logrus` name...